Merchant/App/Pin/PinController.swift

```diff
@@ -21,7 +21,7 @@ class PinController: PinViewController {
         return label
     }()
 
-    var pinView: UIView = {
+    let pinView: UIView = {
```

Comment:
fileprivate please

Reply:
In this commit was the objective to correct the value type only. Talking about the access control I would leave this for another commit which will do only that as there are possibly many more places than just these...